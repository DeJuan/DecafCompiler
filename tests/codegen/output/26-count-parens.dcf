There are 1767263190 ways to balance 20 sets of parentheses