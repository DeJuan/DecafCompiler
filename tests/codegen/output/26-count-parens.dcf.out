There are 4862 ways to balance 9 sets of parentheses
